lib/minify.js

```diff
@@ -125,6 +131,9 @@ function minify(files, options) {
             }
             toplevel = options.parse.toplevel;
         }
+        if (quoted_props) {
+            reserve_quoted_keys(toplevel, quoted_props);
+        }
```

Comment:
Fair enough. Not a big deal in any case. Only the two of us would care about this data.